docker/example/supervisord/Dockerfile

```diff
@@ -10,12 +10,12 @@ LABEL version="${version}"
 ENV DEBIAN_FRONTEND=noninteractive \
     TZ=America/New_York \
     HOME=/root
-RUN (echo $TZ > /etc/timezone) \
- && dpkg-reconfigure tzdata
+RUN (echo $TZ > /etc/timezone)
 
 # Install packages
 RUN apt-get update \
  && apt-get -y upgrade \
+ && apt-get -y install tzdata \
  && apt-get -y install \
```

Comment:
No need to run `apt-get install` twice. Better to insert a line with `    tzdata \` between line 20 and line 21. 

Are you sure installing tzdata after writing `/etc/timezone` on line 13 won't overwrite `/etc/timezone`? I'm wondering if lines 13-15 should be moved to line 25 intact.

Reply:
done